### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by ahmar20**
I updated documentation for compiling the driver.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,26 +27,26 @@ immediatly turn down this git.
...
 - The code has been up-ported to latest OpenWRT trunk using kernel 4.9 or later
-- The code was successfully compiled using Ubuntu 16.04
-- The compiled kernel module is successfully loaded, and network device initiated.
+- The code was successfully compiled using Ubuntu 16.04 and Ubuntu 18.04 on WSL (Windows Subsystem for Linux)
+- The compiled kernel module is successfully loaded, and network device initiated (only for 2.4 GHz devices for now).
 - The driver communicates with a custom hostapd that contains mtlk primitives.
 - There is no IP communication, this is likely related to "SOURCE of RX packet not found!" issue.
 
 ## TODO
+- Investigate the kernel panic while initiating 5 GHz devices
```
Comment (vittorio88) [inline README.md]: Please remove this line, it is not relevant to the WAVE 300 on VR200/VR200v.

File: README.md [hunk 2]
```diff
@@ -27,26 +27,26 @@ immediatly turn down this git.
...
-  error: conflicting types for 'copy_file_range()' which was 
-         introduced by glibc 2.2.7 used in Ubuntu 18.04
+* Build using OpenWRT 18.06.1 on Ubuntu 16.04 LTS or Ubuntu 18.04 for WSL.
 
 
 ### Clone the OpenWRT development environment
 `mkdir ~/src`  
 `cd ~/src`  
 `git clone https://github.com/openwrt/openwrt.git`  
+It is better to switch to current stable release.
```
Comment (vittorio88) [inline README.md]: This line is missing 2 spaces, so the 'git checkout' command is on a separate line.

File: README.md [hunk 3]
```diff
@@ -27,26 +27,26 @@ immediatly turn down this git.
 
 ## Status
 - The code has been up-ported to latest OpenWRT trunk using kernel 4.9 or later
-- The code was successfully compiled using Ubuntu 16.04
-- The compiled kernel module is successfully loaded, and network device initiated.
+- The code was successfully compiled using Ubuntu 16.04 and Ubuntu 18.04 on WSL (Windows Subsystem for Linux)
+- The compiled kernel module is successfully loaded, and network device initiated (only for 2.4 GHz devices for now).
```
Comment (vittorio88) [inline README.md]: This is not correct, also 5 Ghz should work, just not on VR200.

File: README.md [hunk 4]
```diff
@@ -118,12 +118,7 @@ you need to have the following firmware files:
 `/lib/firmware/contr_lm.bin`  
 `/lib/firmware/Progmodel_*.bin`  
 
-You can retrieve these from:
```
Comment (vittorio88) [inline README.md]: Please leave this list of option for finding firmware. The firmware files indicated in lantiq_fw are only useful to a small subset of people, and other people may find their firmwares using these instructions.

**Comment by ahmar20** [comment]
I think changes are done. I am new to GitHub so please bear with me. Thanks

**Pull request closed without merging**